internal/contour/listener.go

```diff
@@ -198,8 +203,8 @@ func (lvc *ListenerVisitorConfig) requestTimeout() time.Duration {
 // minProtocolVersion returns the requested minimum TLS protocol
 // version or envoy_api_v2_auth.TlsParameters_TLSv1_1 if not configured {
 func (lvc *ListenerVisitorConfig) minProtoVersion() envoy_api_v2_auth.TlsParameters_TlsProtocol {
```

Comment:
nit: You renamed `MinimumProtocolVersion` --> `MinimumTLSVersion` but the method is still called `minProtoVersion()`. Change to `minTLSVersion()` to match?

Reply:
Fixed.